routes/api/users.js

```diff
@@ -103,4 +107,56 @@ router.get("/current", verifyToken, async (req, res, next) => {
     next(error);
   }
 });
+
+router.patch(
+  "/avatars",
+  verifyToken,
+  upload.single("avatar"),
+  async (req, res, next) => {
+    try {
+      if (!req.file) {
+        return res.status(400).json({
+          message:
+            "Wrong file type to upload (jpg,png,jpeg,gif allowed) or no avatar uploaded",
```

Comment:
To mozesz podzielić. Czyli sprawdzić jakie są typy plików i wysłać informację że złe jeśli ktoś taki przesłał.